jvm/src/Language/Java.hs

```diff
@@ -416,6 +427,9 @@ class (Interp (Uncurry a) ~ ty, SingI ty, IsReferenceType ty)
       => Reflect a ty where
   reflect :: a -> IO (J ty)
 
+  default reflect :: Coercible a ty => a -> IO (J ty)
+  reflect x = newLocalRef (jobject x)
+
```

Comment:
```haskell
newtype RDD a = RDD (J ('Class "...JavaRDD"))
instance Coercible (RDD a) ...
instance Reify (RDD a) ...
instance Reflect (RDD a) ...
```
as we do now except for the additional `Reify`/`Reflect` instances. Does that not work?